test/view_component/slotable_v2_test.rb

```diff
@@ -300,4 +300,14 @@ def test_slot_with_nested_blocks_content_selectable_false
     # Check shared data through Proc
     refute_selector("div.table div.table__header span", text: "Selectable")
   end
+
+  def test_renders_integers
+    render_inline(SlotsV2Component.new) do |component|
+      component.title { 1 }
+      component.subtitle { '2' }
```

Comment:
To make Rubocop happy:

```suggestion
      component.subtitle { "2" }
```